src/elm/Page/Community.elm

```diff
@@ -65,48 +84,55 @@ subscriptions _ =
 
 type alias Model =
     { date : Maybe Posix
-    , community : LoadStatus
-    , members : List Member
+    , pageStatus : PageStatus
+    , actionId : Maybe Int
     , openObjective : Maybe Int
-    , modalStatus : ModalStatus
-    , invitations : String
-    , symbol : Symbol
+    , claimConfirmationModalStatus : ClaimConfirmationModalStatus
+    , proofs : Maybe Proof
     }
 
 
-initModel : LoggedIn.Model -> Symbol -> Model
-initModel _ symbol =
-    { date = Nothing
-    , community = Loading
-    , members = []
-    , openObjective = Nothing
-    , modalStatus = Closed
-    , invitations = ""
-    , symbol = symbol
-    }
+type Proof
+    = Proof ProofPhotoStatus ProofCode
 
 
-type LoadStatus
+type ProofCode
+    = NoCodeRequired
+    | CodeParts
+        { code : Maybe String
+        , claimTimestamp : Int
+        , secondsAfterClaim : Int
+        , availabilityPeriod : Int
+        }
+
+
+type PageStatus
     = Loading
-    | Loaded Community.Model EditStatus
+    | Loaded Community.Model ActiveSection
     | NotFound
     | Failed (Graphql.Http.Error (Maybe Community.Model))
 
 
+type ActiveSection
+    = Objectives EditStatus
+    | ClaimWithProofs Community.Action
```

Comment:
I wrote this before running the app. After running I can see that the claim with photo is another screen that is rendered in the same elm module. I'm ok with this implementation but initially I imagined it being on a modal. The reason for this is that we will add the ability to do claims with images on the dashboard as well, soon

Reply:
> Having `EditStatus` here is a leftover of a previous interaction

Yep, its only variant is `NoEdit` and it was used everywhere, so I removed it.

To make the type for the page status to be more explicit I also renamed `Objectives` to `ObjectivesAndActions` since there are both presented on the page.

> I imagined it being on a modal <...> claims with images on the dashboard as well, soon

I agree that the "Claim with a photo" dialog looks like a modal but I decided to follow [the design](https://www.figma.com/file/N2lbB9xdO6tFbfg2Is4h7s/claim_with_image?node-id=2%3A0) here. I think it's fine for now to go like this and tweak the implementation after we have a clear vision of how claiming with an image from the Dashboard should be.

Reply:
ok!!